pkg/modulewriter/tfwriter.go

```diff
@@ -331,14 +331,19 @@ func writeVersions(dst string) error {
 	return nil
 }
 
-func printTerraformInstructions(grpPath string, group config.GroupName, printIntergroupWarning bool) {
+func printTerraformInstructions(grpPath string, group config.GroupName, printExportOutputs bool, printImportInputs bool) {
 	printInstructionsPreamble("Terraform", grpPath, string(group))
-	if printIntergroupWarning {
-		fmt.Print(intergroupWarning)
+	fmt.Println()
+	if printImportInputs {
+		fmt.Printf("ghpc import-inputs %s\n", grpPath)
 	}
-	fmt.Printf("  terraform -chdir=%s init\n", grpPath)
-	fmt.Printf("  terraform -chdir=%s validate\n", grpPath)
-	fmt.Printf("  terraform -chdir=%s apply\n\n", grpPath)
+	fmt.Printf("terraform -chdir=%s init\n", grpPath)
```

Comment:
In reference to removing the indent: Can you put an example of how this change looks in the description. I think it is probably fine but am a little worried about readability. I would just like to validate that when it all comes together it is still clear.

Reply:
Added. You'll observe that the indentation has been replaced by a line of vertical whitespace. Very nearly 100% of the time, I cut-and-paste the instructions from screen and having 2 columns of whitespace is a bit hard for "grabbing" the text as well making it harder to search my shell history for previous executions.